src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
```

Comment:
Not sure if you should have a line break here?

Reply:
Ops, I missed that out. Will have it changed